.github/workflows/link-pr-to-issue.yml

```
@@ -0,0 +1,206 @@
+env:
+  LOG_PREFIX: "[PR-Linker]"
+
+jobs:
+  link:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+
+    steps:
+      - name: Validate inputs
+        id: validate
+        run: |
```

actionlint (1.7.12)

[error] "github.event.pull_request.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)